wallet/wallet.go

```diff
@@ -3400,6 +3420,9 @@ func (w *Wallet) StakeInfo(ctx context.Context) (*StakeInfoData, error) {
 
 	var res StakeInfoData
 
+	defer w.lockedOutpointMu.Unlock()
+	w.lockedOutpointMu.Lock()
```

Comment:
shouldn't need the utxo lock when reading stake outputs, these don't have real risk of double spending.